util/neo-nrve-eventhandler.py

```diff
@@ -246,10 +246,10 @@ def send_email(self, subject, body):
                 continue
 
             if self.wallet_needs_recovery:
-                self.recover_wallet()
+                await self.recover_wallet()
                 self.wallet_needs_recovery = False
```

Comment:
This is a similar pattern to the one above, except in this case you are toggling the flag after you do the recovery... My same question from above applies here, except in this case if another thread comes in it will start recovering the wallet at the same time as the other thread, right?

Reply:
Yeah, this is the order we did it previously. I only needed to change the order above because of the (potentially) recursive nature of the execution. There's no such issue here, so the order doesn't matter.

Also, FWIW, this script isn't used anymore; this was used during the token sale to send out refunds. I was just updating this to properly handle the new `asyncio` flavor of the framework...